tests/expansion.in

```diff
@@ -101,6 +101,12 @@ echo $foo[d]
 echo ()[1]
 echo ()[d]
 
+logmsg Percent self
+echo %selfNOT
```

Comment:
Good idea to include this one!

Do we want a test in quotes? Or escaped?

```fish
echo "%self" \%self
```

Reply:
Good idea, I'll add such a test.